userguide/config.toml

```diff
@@ -186,7 +170,7 @@ no = 'Sorry to hear that. Please <a href="https://github.com/google/docsy/issues
 # If you want this feature, but occasionally need to remove the Reading time from a single page, 
 # add "hide_readingtime: true" to the page's front matter
 [params.ui.readingtime]
-enable = true
+enable = false
```

Comment:
I can't remember whether we had this switched on by default...

Reply:
The reading time was switched off. I had switched it on to test the layout together with the tags.